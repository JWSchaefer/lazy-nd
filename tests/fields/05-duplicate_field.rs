use lazy_nd::lazy_nd;
#[lazy_nd(dim = D)]
struct Test<const D: usize> {
    #[scalar(mass:f64)]
    #[scalar(position : f64)]
    field: f64,
    field: bool,
}
fn main() {}
