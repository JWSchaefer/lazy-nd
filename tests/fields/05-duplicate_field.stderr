error: field `field` already declared
 --> $DIR/05-duplicate_field.rs:7:5
  |
7 |     field: bool,
  |     ^^^^^
